source/includes/api-reference/_api-reference.html.md

```diff
@@ -242,7 +338,75 @@ GET /v1/{userName}/{brainName}/sims
 | connected | Count of how many simulators are connected |
 | instances | Array of connected simulators with their status and episode count |
 
+## Simulator Logs
+
+GET log messages from platform-managed simulators.
+
+> HTTP Request
```

Comment:
This is the only place that we point out it's an HTTP Request. Do you think in all other cases we should say HTTP where there is also a Websocket request?